walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,59 +22,87 @@
 
 /** Finds potential meeting times. */
 public final class FindMeetingQuery {
+  public Collection<TimeRange> timesForEveryone;
+  public Collection<TimeRange> timesForConfirmed;
+  public int currentEndTime;
 
   /**
    * Returns available TimeRanges for the {@code request} given {@code events}, a Collection of
    * all events so far. 
```

Comment:
The text: a Collection of all events so far can be removed given we know from the method declaration that its a collection.

Reply:
gotcha, removed!